pkg/tree/tree.go

```diff
@@ -148,8 +148,7 @@ func (t Tree) Empty() bool {
 	return true
 }
 
-// MarshalToTree marshals obj to yaml and then parses the resulting yaml as
-// as Tree.
+// MarshalToTree marshals obj to yaml and then parses the resulting yaml as a Tree.
```

Comment:
We tend to wrap our comments at col 80. I think this was split into multiple lines because of that (it's less than 80 characters on that line but it wouldn't make sense to split "Tree").

Reply:
Ok, I see the fact that code should be wrapped at 80 chars. but still there is a duplication of `as`